Source/ProjectFirma.Web/ScriptsCustom/Maps/ProjectFirmaMaps.js

```diff
@@ -173,6 +199,15 @@ ProjectFirmaMaps.Map.prototype.wmsParams = {
     tiled: true
 };
 
+ProjectFirmaMaps.Map.prototype.westParams = {

```

Comment:
If we don't need that function above, we probably don't need these params either